### PR TITLE
[Merged by Bors] - feat: Prove that a fundamental domain has nonzero volume

### PR DESCRIPTION
**Pull request by xroblot**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Module/Zlattice.lean [hunk 1]
```diff
@@ -317,6 +316,12 @@ protected theorem isAddFundamentalDomain [Finite ι] [MeasurableSpace E] [OpensM
     fun x => exist_unique_vadd_mem_fundamentalDomain b x
 #align zspan.is_add_fundamental_domain Zspan.isAddFundamentalDomain
 
+theorem measure_fundamentalDomain_ne_zero [Finite ι] [MeasurableSpace E] [BorelSpace E]
+    {μ : Measure E} [Measure.IsAddHaarMeasure μ] (hμ : μ ≠ 0) :
```
Comment (sgouezel) [inline Mathlib/Algebra/Module/Zlattice.lean:320]: You should remove h\mu here, as a Haar measure is always nonzero (it gives positive mass to open sets). Same thing below.

Comment (xroblot) [reply Mathlib/Algebra/Module/Zlattice.lean]: I removed the hypothesis from the lemma in `Zlattice.lean`, but I am not sure what you mean by `Same thing below`. Do I need to change also the statement of the lemma in `FundamentalDomain.lean`? If I removed the hypothesis in this lemma, it does not work anymore as it is.

Comment (sgouezel) [reply Mathlib/Algebra/Module/Zlattice.lean]: I read too quickly, what you did is perfect.

**Comment by sgouezel** [comment]
bors d+
Thanks!

**Pull request closed without merging**